metadata.rb

```diff
@@ -1,10 +1,10 @@
 name             'nsis'
 maintainer       'Ketan Padegaonkar'
 maintainer_email 'ketanpadegaonkar@gmail.com'
-license          'MIT'
+license          'Apache v2.0'
```

Comment:
That is supposed to be `Apache-2.0` which is the OSI approved code :) The rest looks ok and I'll merge it.